Classes/UNAlertView.swift

```diff
@@ -210,8 +211,17 @@ final public class UNAlertView: UIView {
     func buttonTapped(btn:UNAlertButton) {
         
         btn.action()
+        if (autoDismiss) {
+            dismiss()
+        }
+    }
+    
+    func dismissAlert() {
         dismiss()
     }
+    func setAutoDismiss(autoStatus: Bool) {
+        autoDismiss = autoStatus
+    }
     
```

Comment:
Currently dismiss is private

`private func dismiss() {`

I felt changing an existing method signature was beyond the scope of a minor PR but I agree, making this public is the easier answer. 

My function 'dismissAlert()' is just a work-around to support that private method, just in case it was made private for a specific reason.